docs/src/scss/_tables.scss

```diff
@@ -13,10 +13,11 @@ table {
     vertical-align: top;
     border: none;
     border-right: $table-border-width solid $table-border-color;
+    border-left: $table-border-width solid $table-border-color;
     vertical-align: middle;
-    &:last-child {
-      border: none;
-    }
+    // &:last-child {
+    //   border: none;
+    // }
   }
```

Comment:
Are the above changes in this file needed?

Reply:
Not for this file Vishnu. Hence removed it for this file and not from the CSS.